src/services/services.ts

```diff
@@ -1950,6 +1950,10 @@ namespace ts {
             return OrganizeImports.organizeImports(sourceFile, formatContext, host, program, preferences);
         }
 
+        function renameFile(oldFilePath: string, newFilePath: string, formatOptions: FormatCodeSettings): ReadonlyArray<FileTextChanges> {

```

Comment:
the name makes me think there is will rename the file as well.. how about `getEditsForFileRename`